language-server/modules/langserver-core/src/test/resources/completion/statement_context/config/foreach_stmt_ctx_config19.json

```diff
@@ -60,11 +60,11 @@
     {
       "label": "filter(function () func)",
       "kind": "Function",
-      "detail": "stream<any|ErrorType, ErrorType?>",
+      "detail": "stream<any|error, error?>",
```

Comment:
This change means this change has broken the current type ref checks for error. This change may be due to the above issue I pointed.

Reply:
Agreed with @pubudu91. This shouldn't be changed

Reply:
Seems like the one suggested earlier was wrong. The `filter()` function in streams has `stream<Type,CompletionType>` as return type and those type params represent subtypes `any|error` and `error` respectively.